.travis.yml

```diff
@@ -55,11 +55,9 @@ before_install:
     - bash $build_path/install_miniconda.sh -b -p $build_path/miniconda
     - source $build_path/miniconda/bin/activate $build_path/miniconda
     - conda update --yes --all 'python<=2.7.12'
-    - conda config --add channels https://conda.binstar.org/cdeepakroy
 
 install:
-    # https://github.com/pypa/pip/issues/2751
-    - conda install --yes 'setuptools>=19.4' ctk-cli==1.3.1
+    - pip install --upgrade 'git+https://github.com/cdeepakroy/ctk-cli'
```

Comment:
When that occurs, we'll be able to change this.  For now, I had to switch away from conda.binstar.org, because their ssl certificate has expired.